src/components/goal/Goal.tsx

```diff
@@ -3,13 +3,71 @@ import Subgoals from "../subgoal/Subgoal";
 import { trpc } from "@/client/lib/trpc";
 import { Goal } from "@/types/global";
 import $goal from "./Goal.module.css";
+import { useState } from "react";
+import Link from "next/link";
+
+type ControlCaratProps = {
+  direction: "down" | "right" | "left" | "up";
```

Comment:
Not currently, but when I was doing this I was first taking this on I was adding the capability of clicking on the carat to display subgoals/benchmarks below, which would require the carat pointing in another direction. We can probably abandon that, but this might also be the sort of reusable component that we want to create and save as part of the design systems issue. I'm not at all married to it, but wanted to see what (if anything) should be saved before I toss it.